.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
can you please rollback this file?
also please consider setting up global ignore files https://gist.github.com/subfuzion/db7f57fff2fb6998a16c